nix/overlays/haskell-packages.nix

```diff
@@ -17,6 +17,33 @@ let
       #
       # To get the sha256:
       #   nix-prefetch-url --unpack https://hackage.haskell.org/package/protolude-0.3.0/protolude-0.3.0.tar.gz
+
+      # TODO: We need to patch upstream for unbreaking hasql-dynamic-statements, hasql-implicits, ptr
+      hasql-dynamic-statements =
+        self.haskell.lib.dontCheck (prev.callHackageDirect
+          {
+            pkg = "hasql-dynamic-statements";
+            ver = "0.3.1";
+            sha256 = "1zjv91xlfkyxwq6mhzj7rsfm4kjvs9ygkgbl6jbbg19jihcn2kiy";
+          }
+          { }
+        );
```

Comment:
@monacoremo This might be one for you. Whenever you have the time :)

Btw, I learned the `lib.dontCheck` trick thanks to this post: https://discourse.nixos.org/t/nix-haskell-development-2020/6170.